Library/Environment.swift

```diff
@@ -46,6 +46,11 @@ public struct Environment {
   /// The current device running the app.
   public let device: UIDeviceType
 
+  /// Returns the current environment type
+  public var environmentType: EnvironmentType {
+    return self.apiService.serverConfig.environment
```

Comment:
I got tired of always typing this out when trying to determine the current environment type. Now you can just do `AppEnvironment.current.environmentType` 😸